src/main/resources/application.properties

```diff
@@ -33,8 +33,8 @@ ehrRepoAuthKey=${EHR_TRANSFER_SERVICE_AUTHORIZATION_KEYS_FOR_EHR_REPO}
 gp2gpMessengerUrl=${EHR_TRANSFER_SERVICE_GP2GP_MESSENGER_URL}
 gp2gpMessengerAuthKey=${EHR_TRANSFER_SERVICE_AUTHORIZATION_KEYS_FOR_GP2GP_MESSENGER}
 processingPeriodMilliseconds=10000
-ehrResponsePollPeriodMilliseconds=10000
-ehrResponsePollLimit=60
+ehrResponsePollPeriodMilliseconds=1000
```

Comment:
`ehrTransferFinalisedPollPeriodMilliseconds`?

Reply:
I like this, will update.

Reply:
On a counter argument - I'd say perhaps keeping it simple could be better. Taking a look from an outsiders perspective `ehrTransferFinalisedPollPeriodMilliseconds` could seem quite confusing in my opinion.

How about:

1. `timeoutPollPeriodMilliseconds` - as there is only one piece of timeout logic present?